templates/vue/src/components/TapestryMap.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+
+  <div style="height: 800px; width: 100%">
+    <div style="height: 150px overflow: auto;">
+      <p>First marker is placed at {{ withPopup.lat }}, {{ withPopup.lng }}</p>
+      <p> The bounds are: {{ mapBounds }} </p>
+      <p> Settings are: {{this.settings}} </p>
```

Comment:
You never use `.this` in the template. The template is bound to `this` already so the variables you use in it are assumed to be properties of `this` already. So this should be just `settings`.